compass_sdk/constants.py

```diff
@@ -33,3 +33,5 @@
     "date",
     "authors",
 ]
+
+UUID_NAMESPACE = "00000000-0000-0000-0000-000000000000"
```

Comment:
This seems a big automagic, can you explain how this constant is signifcant?

Reply:
Got it from the uuid creation examples. It can be deleted if the result is the same without specifying the namespace.

Reply:
Ah, read into it in the examples, and passing a namespace like this also designates this string as "null" - that seems like a worthwhile special case to have